client/src/main/java/com/alibaba/nacos/client/identify/CredentialWatcher.java

```diff
@@ -187,26 +209,26 @@ private void loadCredential(boolean init) {
                 SPAS_LOGGER.info("[{}] Load credential file {}", appName, propertyPath);
             }
             
-            if (!IdentifyConstants.DOCKER_CREDENTIAL_PATH.equals(propertyPath)) {
-                if (properties.containsKey(IdentifyConstants.ACCESS_KEY)) {
-                    accessKey = properties.getProperty(IdentifyConstants.ACCESS_KEY);
+            if (!DOCKER_CREDENTIAL_PATH.equals(propertyPath)) {
+                if (properties.containsKey(ACCESS_KEY)) {
```

Comment:
For some constants which may get from properties. They should be still use IdentityConstants.

The reason is users may use the IdentityConstants to init properties key before they call `Factory`

Reply:
Alright.